ci/run_tests_test.go

```diff
@@ -195,18 +196,22 @@ printf "RandomSecret: {{ .RandomSecret }}\n"
 		t.Fatal(err)
 	}
 	if submission.Status == qf.Submission_APPROVED {
-		t.Error("Submission must not be auto approved")
+		t.Error("submission must not be auto approved")
```

Comment:
It is okay and expected to use capitalization for `t.Error` and `t.Fatal` etc. 

It is the `return fmt.Errorf()` and `return errors.New()` that should be lowercase; this is because errors returned up from a function call can be nested in a final printing of the error... and then we don't want lots of capitalization.